docs/pages/style/design.md

```diff
@@ -71,26 +71,26 @@ Here’s what that looks like for recent CA.gov initiatives such as covid19.ca.g
 </div>
 
 
-### Seondary color 
+### Secondary color 
 <div class="grid-container">
   <div class="text-center p-t-3">
     <div class="bg-secondary-color radius-round w-7 h-7 m-x-a"></div>
-    <p class="font-size-1 m-t-2 m-b-0 text-bold">Seondary color</p>
+    <p class="font-size-1 m-t-2 m-b-0 text-bold">Secondary color</p>
     <p class="font-size-1">#FF8000</p>
```

Comment:
Caught one more hex code here (#FF8000) that should be lowercase #ff8000

Reply:
@kmbrlygl Fixed that one